pennylane/measurements/mid_measure.py

```diff
@@ -97,11 +105,11 @@ def circuit(x, y):
             qml.RY(y, wires=1)
             m0 = qml.measure(1)
             return (
-                qml.expval(m0), qml.var(m0), qml.probs(op=m0), qml.counts(op=m0), qml.sample(m0)
+                qml.sample(m0), qml.expval(m0), qml.var(m0), qml.probs(op=m0), qml.counts(op=m0),
```

Comment:
I also moved the `qml.sample` return to the front so that the 0s and 1s are more visible